include/multiphase/VCTwoFluidStaggeredStokesBoxRelaxationFACOperator.h

```diff
@@ -268,10 +268,10 @@ class VCTwoFluidStaggeredStokesBoxRelaxationFACOperator : public IBTK::FACPrecon
     double d_nu_s = std::numeric_limits<double>::quiet_NaN();
 
     SAMRAI::tbox::Pointer<SAMRAI::hier::PatchHierarchy<NDIM>> d_hierarchy; // Reference patch hierarchy
-    std::vector<SAMRAI::solv::RobinBcCoefStrategy<NDIM>*> d_un_bc_coefs;
-    std::vector<SAMRAI::solv::RobinBcCoefStrategy<NDIM>*> d_us_bc_coefs;
     std::unique_ptr<SAMRAI::solv::RobinBcCoefStrategy<NDIM>> d_default_un_bc_coef, d_default_us_bc_coef,
         d_default_P_bc_coef, d_default_thn_bc_coef;
+    std::vector<SAMRAI::solv::RobinBcCoefStrategy<NDIM>*> d_un_bc_coefs;
+    std::vector<SAMRAI::solv::RobinBcCoefStrategy<NDIM>*> d_us_bc_coefs;
```

Comment:
Note that this is one reason to pay attention to compiler warnings in #70. These needed to be initialized in a different order than specified in the header.